### PR TITLE
[Sizer]: Overflow issue fix on homscreen

### LINKED ISSUE #15: Overflow issue on homescreen.
Getting a overflow issue on Homescreen due to the 2 buttons.

overflow

I would like to work on this, please assign it to me.

### PR DESCRIPTION
**Pull request by Aadeesh11**
Fixes #15 

Used package sizer to fix the overflows and not used constant height and width values, but dynamically calculated ones from sizer.

Screenshot 2021-12-27 at 1 03 19 AM

### CODE REVIEW

**Comment by Literal-Eval** [comment]
Instead of using git dependency, use pub dependency. And that issue was fixed anyway, so be sure to visit that fix to be consistent.

**Comment by Literal-Eval** [comment]
Instead of using git dependency, use pub dependency instead. And that issue was fixed anyway, so be sure to visit that fix to be consistent.

**Comment by Aadeesh11** [comment]
> Instead of using git dependency, use pub dependency. And that issue was fixed anyway, so be sure to visit that fix to be consistent.

Yes, i had started using the git one around a month ago so forgot about it. Will use the pub one from now on.

**Comment by Aadeesh11** [comment]
@nb9960 Will this fix be merged, or does it need some changes?

File: lib/main.dart [hunk 1]
```diff
@@ -9,6 +9,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_config/flutter_config.dart';
 import 'package:hive/hive.dart';
+import 'package:sizer/sizer.dart';
```
Comment (nb9960) [inline lib/main.dart:12]: Remove this duplicate import

**Pull request merged**